src/module_PI/Raspberry_PI/JUnitTests/EmailNotifierTest.java

```diff
@@ -80,4 +80,16 @@ public void testSetRecipientEmailAddress_InvalidEmailAddress(){
 		expectedException.expectMessage("notanemailaddress is not a valid email address");
 		testEmailNotifier.setRecipientEmailAddress("notanemailaddress");
 	}
+	
+	@Test
+	public void testRemoveAttachment(){
+		testEmailNotifier.removeAttachment();
```

Comment:
If the default state of an emailNotifier is to have no attachment then you should add an attachment and then remove it in order to prove your method is working. Otherwise, your method might be completely broken but you'd never know it because your emailNotifier is already sitting in the correct state.